jbrotli/src/main/java/org/meteogroup/jbrotli/io/BrotliInputStream.java

```diff
@@ -99,7 +100,12 @@ private void readChunkFromInput() throws IOException {
     while (!isEndOfInputStream && (uncompressedBufferPosition == 0 || brotliStreamDeCompressor.needsMoreInput())) {
       final int inLength = inputStream.read(in);
       if (inLength > 0) {
-        uncompressedBufferPosition += brotliStreamDeCompressor.deCompress(in, 0, inLength, uncompressedBuffer, uncompressedBufferPosition, uncompressedBuffer.length - uncompressedBufferPosition);
+        int[] lengths = brotliStreamDeCompressor.deCompress(in, 0, inLength, uncompressedBuffer, uncompressedBufferPosition, uncompressedBuffer.length - uncompressedBufferPosition);
+        if (lengths[0] < inLength) {
+          // TODO: handle this case
+          throw new IOException("Compressor did not consume all input");
```

Comment:
This is temporary, until it can handle the case where Brotli doesn't consume all of the input. Right now it throws IOException because it is an unrecoverable problem.